src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewController.swift

```diff
@@ -119,13 +147,22 @@ class FooterViewController: UIViewController {
 		subscription.forEach { $0.cancel() }
 		subscription.removeAll()
 		
-		NSLayoutConstraint.deactivate(buttonConstraints)
-		buttonConstraints.removeAll()
+		primaryButton.isHidden = true
+		secondaryButton.isHidden = true
```

Comment:
These two lines are confusing. Could you please add a comment, why this is happening.

Reply:
Done